include/ebpf_windows.h

```diff
@@ -30,6 +30,7 @@ typedef uint8_t GUID[16];
 #define EBPF_PROGRAM_DATA_PLATFORM_SPECIFIC_DATA L"PlatformSpecificData"
 #define EBPF_PROGRAM_DATA_PRIVELEGED L"IsPrivileged"
 #define EBPF_PROGRAM_DATA_BPF_PROG_TYPE L"BpfProgType"
+#define EBPF_PROGRAM_DATA_BPF_ATTACH_TYPE L"BpfAttachType"
```

Comment:
Macro for `BpfAttachType` is already defined on line 26.

Reply:
Should there be separate `PROGRAM_DATA` and `SECTION_DATA` constants then? The function that had the bug was `_load_section_data_information` - so would it make more sense to have `SECTION_DATA` macros as well?

Reply:
Since both have the same string, I think we should not define them twice. Alternatively, we can rename the string to a more generic name to be say "EBPF_DATA_BPF_ATTACH_TYPE" / "EBPF_DATA_BPF_PROG_TYPE" which can then be used for both program and section data?